packages/common/src/directives/ng_optimized_image/ng_optimized_image.ts

```diff
@@ -280,10 +280,12 @@ export class NgOptimizedImage implements OnInit, OnChanges, OnDestroy {
   @Input({transform: numberAttribute}) height: number|undefined;
 
   /**
-   * The desired loading behavior (lazy, eager, or auto).
+   * The desired loading behavior (lazy, eager, or auto). Defaults to `lazy`,
+   * which is recommended for most images.
    *
-   * Setting images as loading='eager' or loading='auto' marks them
-   * as non-priority images. Avoid changing this input for priority images.
+   * Warning: Setting images as loading='eager' or loading='auto' marks them
+   * as non-priority images and can hurt loading performance. Avoid changing
+   * this input for priority images.
```

Comment:
Maybe add something about what to do instead? e.g.
```suggestion
   * as non-priority images and can hurt loading performance. For priority images (i.e. images that might be the [LCP element]()), try adding the ["priority" attribute]() instead of changing the "loading" value.
```